types/victory/index.d.ts

```diff
@@ -513,6 +513,7 @@ declare module "victory" {
     /**
      * Default theme
      */
+    grayscale: VictoryThemeDefinition;
```

Comment:
grayscale is the default: https://formidable.com/open-source/victory/docs/victory-theme/